packages/twenty-server/src/database/typeorm-seeds/core/workspaces.ts

```diff
@@ -1,14 +1,52 @@
 import { DataSource } from 'typeorm';
 
+import { Workspace } from 'src/core/workspace/workspace.entity';
+
 const tableName = 'workspace';
 
-export const SeedWorkspaceId = '20202020-1c25-4d02-bf25-6aeccf7ea419';
+export const SeedAppleWorkspaceId = '20202020-1c25-4d02-bf25-6aeccf7ea419';
+export const SeedTwentyWorkspaceId = '3b8e6458-5fc1-4e63-8563-008ccddaa6db';
 
 export const seedWorkspaces = async (
   workspaceDataSource: DataSource,
   schemaName: string,
   workspaceId: string,
 ) => {
+  let workspaces: Pick<
+    Workspace,
+    | 'id'
+    | 'displayName'
+    | 'domainName'
+    | 'inviteHash'
+    | 'logo'
+    | 'subscriptionStatus'
+  >[] = [];
+
+  if (workspaceId === SeedAppleWorkspaceId) {
```

Comment:
@AdityaPimpalkar Maybe we should have a map above the function so it's more clean, and works properly if we add more workspaces later like:
```typescript
const workspaces: {
  [key: string]:  Pick<
     Workspace,
     | 'id'
     | 'displayName'
     | 'domainName'
     | 'inviteHash'
     | 'logo'
     | 'subscriptionStatus'
   >;
} = {
  [SeedAppleWorkspaceId]: { /* ... */ },
  [SeedTwentyWorkspaceId]: { /* ... */ }
}
```